packages/cmf/src/cmfConnect.js

```diff
@@ -121,43 +121,52 @@ export default function cmfConnect({
 			};
 			static WrappedComponent = WrappedComponent;
 
+			constructor(props, context) {
+				super(props, context);
+				this.dispatchActionCreator = this.dispatchActionCreator.bind(this);
+			}
+
 			componentDidMount() {
 				initState(this.props);
 				if (this.props.didMountActionCreator) {
-					this.props.dispatchActionCreator(
+					this.dispatchActionCreator(
 						this.props.didMountActionCreator,
 						null,
 						this.props,
-						this.context,
 					);
 				}
 			}
 
 			componentWillUnmount() {
 				if (this.props.willUnmountActionCreator) {
-					this.props.dispatchActionCreator(
+					this.dispatchActionCreator(
 						this.props.willUnmountActionCreator,
 						null,
 						this.props,
-						this.context
 					);
 				}
 				if (!keepComponentState) {
 					this.props.deleteState();
 				}
 			}
 
+			dispatchActionCreator(actionCreatorId, event, data, context) {
+				const myContext = context || this.context;
```

Comment:
As discussed together, this should merge the cmf context with the user context to not break cmf internal process which need the registry in context.